README.md

```diff
@@ -1,12 +1,44 @@
-# OpenTelemetry Instrumentation for Java
-
-## Join the discussions!
-
-* Watch this repo :eye:
-* Join the [Gitter channel](https://gitter.im/open-telemetry/opentelemetry-java-instrumentation)
-* Join the [weekly meeting](https://github.com/open-telemetry/community#java-instrumentation)
-
-## Introduction
+---
+
+<p align="center">
+  <strong>
+    <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation#getting-started">Getting Started<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://github.com/open-telemetry/community#special-interest-groups">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-java-instrumentation">Getting In Touch<a/>
+  </strong>
+</p>
+
+<p align="center">
+  <a href="https://goreportcard.com/report/github.com/open-telemetry/opentelemetry-java-instrumentation">
+    <img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
+  <a href="https://circleci.com/gh/open-telemetry/opentelemetry-java-instrumentation">
+    <img alt="Build Status" src="https://img.shields.io/circleci/build/github/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
+  <a href="https://codecov.io/gh/open-telemetry/opentelemetry-java-instrumentation/branch/master/">
+    <img alt="Codecov Status" src="https://img.shields.io/codecov/c/github/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
+  <a href="releases">
```

Comment:
Wouldn't https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases be safer? Absolute URL will work both from the repo view and the file view